v0x01/utils.py

```diff
@@ -1,18 +1,17 @@
-"""Utilities module for of_core OpenFlow v0x01 operations"""
-from kytos.core import KytosEvent
-from kytos.core.interface import Interface
-
-from napps.kytos.of_core.utils import emit_message_out
-
+"""Utilities module for of_core OpenFlow v0x01 operations."""
 from pyof.v0x01.controller2switch.common import ConfigFlag, FlowStatsRequest
 from pyof.v0x01.controller2switch.set_config import SetConfig
 from pyof.v0x01.controller2switch.stats_request import StatsRequest, StatsType
 from pyof.v0x01.symmetric.echo_request import EchoRequest
 from pyof.v0x01.symmetric.hello import Hello
 
+from kytos.core import KytosEvent
+from kytos.core.interface import Interface
+from napps.kytos.of_core.utils import emit_message_out
+
 
 def update_flow_list(controller, switch):
-    """Method responsible for request stats of flow to switches.
+    """Request stats of flow to switches.
```

Comment:
Please replace with the more correct phrasing "Request flow stats from switches."

Reply:
I will change here.